code/game/jobs/job/supervisor.dm

```diff
@@ -262,9 +263,12 @@ GLOBAL_DATUM_INIT(captain_announcement, /datum/announcement/minor, new(do_newsca
 	glasses = /obj/item/clothing/glasses/hud/security/sunglasses/read_only
 	id = /obj/item/card/id/security
 	l_pocket = /obj/item/laser_pointer
-	r_pocket = /obj/item/flash
+	r_pocket = /obj/item/clothing/accessory/lawyers_badge
```

Comment:
**OBJECTION**

Shouldnt this spawn on the lawyers jumpsuit itself?

Reply:
This'd be annoying if you start with a custom jumpsuit in loadout, imo its fine to be in inventory